docs/quickstart.md

```diff
@@ -293,6 +293,453 @@ hive>
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 09:15 min
+[INFO] Finished at: 2018-09-10T17:47:37-07:00
+[INFO] Final Memory: 236M/1848M
+[INFO] ------------------------------------------------------------------------
+```
+
+#### Bringing up Demo Cluster
+
+The next step is to run the docker compose script and setup configs for bringing up the cluster
+
+```
+cd docker
+./setup_demo.sh
+Stopping spark-worker-1            ... done
+Stopping hiveserver                ... done
+Stopping hivemetastore             ... done
+Stopping historyserver             ... done
+.......
+......
+Creating network "hudi_demo" with the default driver
+Creating hive-metastore-postgresql ... done
+Creating namenode                  ... done
+Creating zookeeper                 ... done
+Creating kafkabroker               ... done
+Creating hivemetastore             ... done
+Creating historyserver             ... done
+Creating hiveserver                ... done
+Creating datanode1                 ... done
+Creating spark-master              ... done
+Creating adhoc-1                   ... done
+Creating adhoc-2                   ... done
+Creating spark-worker-1            ... done
+Copying spark default config and setting up configs
+Copying spark default config and setting up configs
+Copying spark default config and setting up configs
+varadarb-C02SG7Q3G8WP:docker varadarb$ docker ps
+```
+
+At this point, the docker cluster will be up and running. The demo cluster brings up the following services
+
+   * HDFS Services (NameNode, DataNode)
+   * Spark Master and Worker
+   * Hive Services (Metastore, HiveServer2 along with PostgresDB)
+   * Kafka Broker and a Zookeeper Node (Kakfa will be used as upstream source for the demo) 
+   * Adhoc containers to run Hudi/Hive CLI commands
+
+### Demo
+
+Stock Tracker data will be used to showcase both different Hudi Views and the effects of Compaction. 
+
+Take a look at the director `docker/demo/data`. There are 2 batches of stock data - each at 1 minute granularity. 
+The first batch contains stocker tracker data for some stock symbols during the first hour of trading window 
+(9:30 a.m to 10:30 a.m). The second batch contains tracker data for next 30 mins (10:30 - 11 a.m). Hudi will
+be used to ingest these batches to a dataset which will contain the latest stock tracker data at hour level granularity.
+The batches are windowed intentionally so that the second batch contains updates to some of the rows in the first batch.
+
+#### Step 1 : Publish the first batch to Kafka
+
+Upload the first batch to Kafka topic 'stock ticks'
+
+```
+cat docker/demo/data/batch_1.json | kafkacat -b kafkabroker -t stock_ticks -P 
+
+To check if the new topic shows up, use
+kafkacat -b kafkabroker -L -J | jq .
```

Comment:
Let's add a sample output here to ensure what people should compare against..

Reply:
Fixed.